docs/setup-cluster/k8s/setup-eks-cluster.rst

```diff
@@ -206,7 +206,7 @@ command to install the plugin:
 .. code:: bash
 
    # Deploy a DaemonSet that enables the GPUs.
-   kubectl apply -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/master/nvidia-device-plugin.yml
+   kubectl apply -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v0.15.0/deployments/static/nvidia-device-plugin.yml
```

Comment:
Maybe just leave it to use main but update the path?

Does this upgrade often? I wouldn't want us to recommend a really old device plugin. In some sense the link breaking did its job since we updated it and kept it current.